### ISSUE TITLE
fabric.Canvas.loadFromJSON

### ISSUE THREAD

**Issue by TimHarker** [fabricjs/fabric.js#1075]
I'd like to do something "similar to the below", but for fabric.Canvas.loadFromJSON?  Any help would be awesome.  Basically, I'd like to intercept/interact with the items AS they are added to the canvas and while loadFromJSON is running.

```
fabric.Image.fromURL = function (d, f, e) {
    var c = fabric.document.createElement("img");
    c.onload = function () {
        if (f) { f(new fabric.Image(c, e)) }
        c = c.onload = null
    };
    c.setAttribute('crossOrigin', 'anonymous');
    c.src = d;
};
```

I can use the loadFromJSON callback function to successfully interact with the items AFTER they've been added but I'd like to interact with the items AS they are added to the canvas.

Basically, I need to add the crossOrigin attribute BEFORE loadFromJSON loads any images to the canvas - similar to what I do above when not using loadFromJSON.

I hope I explained myself clearly.  I appreciate any and everyone's help.

Cheers.  And HAPPY NEW YEAR!
